Update governance-json-ld.md (#230)
bcgov/digital-trust-toolkit: docs/governance/mining/major-mines-office/governance.md

# 2. Controlled Documents

## 2.1. Glossary
 - [ToIP Core Glossary](https://glossary.trustoverip.org/)
 - [UNTP Conformity Credential ](https://jargon.sh/user/unece/ConformityCredential/v/0.5.0/artefacts/readme/render)

>  - [Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/#json-ld)

Reviewer: ```suggestion
 - [Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0)
```

The operations and processes of the Major Mines Office are governed by [the Mines Act](https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01). 

## 2.5. Business Requirements

> Technical requirements include both Json-LD and Anoncred formats to enable interoperability with multiple formats:

Reviewer: ```suggestion
Technical requirements include both Verifiable Credential and AnonCred formats to enable interoperability with multiple standards:
```

> The Json-LD credential format is designed to meet W3C VCDM 2.0 requirements and is modeled for compliance with the UN Transparency Protocol Digital Conformity Credential.

Reviewer: ```suggestion
The Verifiable Credential format is designed to meet compliance with the UN Transparency Protocol Digital Conformity Credential.
```

> The Anoncreds credential format enables the exchange of credential data between digital wallets once a trusted connection is established.

Reviewer: ```suggestion
The AnonCreds format enables the exchange of claims between digital wallets through an established and trusted connection.
```

2. After selecting the specific mine from the MineSpace home screen, The Mine Manager MUST request a single-use invitation link for the required mining company (based on the mine site selected). 
3. MineSpace MUST present the connection invitation link (URL) to the Mine Manager.
4. The Mine Manager MUST use the connection invitation link provided by MineSpace within their organizational wallet to establish a secure connection.

## 2.5.2. Major Mines Operating Permit Issuance - Anoncreds Exchange

1.	The Mine Manager MUST look up the authorizations and permits for the selected mine within MineSpace to display the request button for an operating permit.
1. The Mine Manager MUST select the permit that is desired as a credential.
1.	Traction SHALL offer the credential to the Company’s organizational wallet.
1.	The organizational wallet MUST display offer details to the Mine Manager. 
1.	The Mine Manager MUST review the credential offer.
1.	The Mine Manager MUST either Accept/Decline the offer. 
1.	Traction MUST receive a problem report if the credential is declined.
1.	Traction MUST notify Core if the offer is declined.

> ## 2.5.3. Publishing Mining Permit Data to OrgBook BC - JSON-LD Credentials

Reviewer: ```suggestion
## 2.5.3. Publishing Mining Permit Data to OrgBook BC - Verifiable Credential
```

> Automated process takes credential data from MDS and creates JSON-LD credentials that are published to Orgbook BC scheduled for weekly deployments. Data MUST meet the following requirements in order to be published:

Reviewer: ```suggestion
Automated process takes data from MDS and creates Verifiable Credential records that are published to Orgbook BC scheduled for weekly deployments. Data MUST meet the following requirements in order to be published:
```

> 1. Party is related to an Orgbook BC entity

Author: ```suggestion
1. The business entity, issuedToParty, is an Orgbook BC entity
```

> 2. Party is assigned as a permittee to the permit on the date of the amendment issued

Author: ```suggestion
2. The business entity, issuedToParty, is assigned as a permittee to the permit on the date of the amendment issued
```

3. The permit is associated with a major mine
4. The permit has status "open"
5. The permit is not an Exploration Permit

## 2.6. Technical Requirements

> Technical requirements include both JSON-LD and Anoncreds verifiable credential formats to enable interoperability of credential data. 

Reviewer: ```suggestion
Technical requirements include both Verifiable Credential and Anoncreds digital credential formats to enable interoperability of claimed data. 
```

>  - [JSON-LD credentials](https://www.w3.org/TR/json-ld11/) are compliant with [Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/#json-ld) and published to [OrgBook BC](https://orgbook.gov.bc.ca/about/orgbook-data)

Reviewer: ```suggestion
 - [Verifiable Credential](https://www.w3.org/TR/vc-data-model-2.0/) records are published to [OrgBook BC](https://orgbook.gov.bc.ca/about/orgbook-data)
```

>    - extends the Digital Conformity Credential of [UNTP, version 0.5.0](https://test.uncefact.org/vocabulary/untp/dcc/0.5.0/)

Reviewer: ```suggestion
   - intended to extend the Digital Conformity Credential of [UNTP, version 0.5.0](https://test.uncefact.org/vocabulary/untp/dcc/0.5.0/)
```

>  -  [AnonCreds credentials](https://wiki.hyperledger.org/display/anoncreds) enable the exchange of credential data between digital wallets once a trusted connection is established. The Major Mines Office uses a Hyperledger Aries-compatible business wallet, therefore mining operators MUST also use a Hyperledger Aries-compatible business wallet solution:

Reviewer: ```suggestion
 -  [AnonCreds credentials](https://wiki.hyperledger.org/display/anoncreds) enable the exchange of claims between digital wallets through an established and trusted connection. The Major Mines Office uses a Hyperledger Aries-compatible business wallet, therefore mining operators MUST also use a Hyperledger Aries-compatible business wallet solution:
```

## 2.9. Legal Agreements

Issuance of a B.C. Mines Act Permit credential in the anoncreds format requires the following terms of use:
 - [Mines Act Permit Credential Terms of Use](../bc-mines-act-permit/1.1.1/2023-11-23%20MAP%20TOU%20V1%20Final.pdf)

> Publishing B.C. Mines Act Permit credential data in JSON-LD format is provided under [Access Only Data Terms and Conditions](https://bcgov.github.io/data-publication/pages/faq.html#data-publication-licensing-options---open-dataaccess-only).

Reviewer: ```suggestion
Publishing B.C. Mines Act Permit data in the Verifiable Credential format is provided under [Access Only Data Terms and Conditions](https://bcgov.github.io/data-publication/pages/faq.html#data-publication-licensing-options---open-dataaccess-only).
```
